.github/workflows/build.yml

```diff
@@ -13,6 +13,8 @@ jobs:
 
     strategy:
       matrix:
+        package:
+          - flutter_hooks
```

Comment:
leftover?

Reply:
No it's needed :) on next PR I'll add flutter_mobx_hooks in the matrix to have some CI